linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_name_format.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Type of hostname to record the audit event'
+
+description: 'Type of hostname to record the audit event'
+
+type: string
+
+interactive: false
+
+options:
```

Comment:
According to man:
```It has the following choices: none, hostname, fqd, numeric, and user.```

It would be good to also include `user` option here.

Reply:
Agree

Reply:
One thing it needs to be improved here is actually the OVAL check that only accepts the three options mandated by STIG. Selecting any other option than those three would result in a fail, so we need to expand more this code. There are similar rules that use variables as a regex and from that regex they use the first option as the value to remediate for example.

https://github.com/ComplianceAsCode/content/blob/0085a63292f63f606e94d86a78aa5314cdd002a2/linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_disk_full_action/ansible/shared.yml#L11

https://github.com/ComplianceAsCode/content/blob/0085a63292f63f606e94d86a78aa5314cdd002a2/linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_disk_full_action/oval/shared.xml#L32

https://github.com/ComplianceAsCode/content/blob/0085a63292f63f606e94d86a78aa5314cdd002a2/linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var#L24

Reply:
Yes, it would be great to also update the OVAL.

Reply:
I will put this one in draft state for now until I get back to it.